frontend/src/clients/internalApiClient.ts

```diff
@@ -35,6 +35,11 @@ export interface NewSightingRequest {
   speciesId: number;
 }
 
+export interface authenticateLoginResponse {
```

Comment:
Should be capitalised
```suggestion
export interface AuthenticateLoginResponse {
```